mcpi/util.py

```diff
@@ -1,14 +1,14 @@
 import collections
 
 try:
-    str
+    basestring
 except NameError:
-    str = (str, bytes)
+    basestring = (str, bytes)
```

Comment:
Basestring is a Python 2 only construct. If anything, basestring should be removed, not added.